docs/developers/styleguide.rst

```diff
@@ -324,11 +324,13 @@ Quoting
         <a href="{% url 'whatever' %}" class="highlight">
 
 
-
         <!-- Bad -->
+        <a href='{% url "whatever" %}' class='highlight'>
+
+    .. code-block:: text
+
         <a href="{% url "whatever" %}" class="highlight">
         <a href='{% url 'whatever' %}' class='highlight'>
-        <a href='{% url "whatever" %}' class='highlight'>
```

Comment:
Then please put all bad examples in the text code-block instead of spreading them in two separate code-blocks.